drivers/espi/Kconfig.xec

```diff
@@ -191,3 +191,16 @@ config ESPI_PERIPHERAL_KBC_OBE_CBK
 	  Enable KBC OBE callback from OBE ISR
 
 endif #ESPI_XEC_V2 && ESPI_PERIPHERAL_8042_KBC
+
+if ESPI_FLASH_CHANNEL
+
+config ESPI_FLASH_STORAGE
+	bool "ESPI Flash channel storage"
+	default y
```

Comment:
This should not be by default enabled better to add depends on device tree

Reply:
ack'd